.code-samples.meilisearch.yaml

```
@@ -87,3 +87,113 @@ getting_started_front_end_integration_md: |-
   - Other Vue components such as `<ais-configure>` and `<ais-highlight>` are optional. They offer greater control over `instant-meilisearch`'s behavior and appearance
   - The first two`<script src="..">` tags import libraries needed to run `instant-meilisearch` with Vue
   - The third and final `<script>` creates a new Vue instance and instructs it to use `instant-meilisearch`
+
+
+landing_getting_started_with_vue_3: |-
+  // In main.js
+  import { createApp } from "vue";
+  import App from "./App.vue";
+  import InstantSearch from "vue-instantsearch/vue3/es";
+
+  createApp(App).use(InstantSearch).mount("#app");
```

```suggestion
  import { createApp } from 'vue';
  import App from './App.vue';
  import InstantSearch from 'vue-instantsearch/vue3/es';
  import "instantsearch.css/themes/algolia-min.css";
  import "@meilisearch/instant-meilisearch/templates/basic_search.css";

  createApp(App).use(InstantSearch).mount('#app');
```

I added the CSS imports here